go.mod

```diff
@@ -38,10 +38,10 @@ require (
 	github.com/syndtr/goleveldb v1.0.1-0.20220614013038-64ee5596c38a
 	github.com/tyler-smith/go-bip39 v1.1.0
 	github.com/urfave/cli/v2 v2.17.2-0.20221006022127-8f469abc00aa
-	golang.org/x/crypto v0.1.0
-	golang.org/x/sync v0.1.0
-	golang.org/x/sys v0.8.0
-	golang.org/x/text v0.7.0
+	golang.org/x/crypto v0.10.0
+	golang.org/x/sync v0.3.0
+	golang.org/x/sys v0.9.0
+	golang.org/x/text v0.10.0
```

Comment:
can we reset the changes to `go.mod` and `go.sum` ? 

We usually cautiously update dependencies and try to avoid coupling it to unrelated changes

Reply:
Sure thing, I wasn't entirely sure how those were updated anyways